newdle/client/src/components/CreateNewdle.js

```diff
@@ -52,29 +43,104 @@ function TimeSlotsPage() {
+            Back
+            <Icon name="caret left" />
+          </Button>
+          <Button color="violet" icon labelPosition="right" onClick={() => dispatch(setStep(3))}>
+            Next step
+            <Icon name="caret right" />
+          </Button>
+        </div>
+      </Grid.Row>
+    </Grid>
+  );
+}
+
+async function createNewdle() {
+  const results = await client.createNewdle();
+  return results;
+}
+
+function FinalizePage() {
+  const [successMessage, setSuccessMessage] = useState(false);
+  const [newdle, setNewdle] = useState(null);
+  const dispatch = useDispatch();
+  return (
+    <Container text>
+      {!successMessage && !newdle ? (
+        <>
+          <Header as="h2" className={styles['input-title']}>
+            Title of your meeting
+          </Header>
+          <Input
+            className={styles['title-input']}
+            focus
+            onChange={(_, data) => dispatch(setTitle(data.value))}
+          />
+          <div className={styles['attention-message']}>
+            <Header as="h3" className={styles['header']}>
+              Attention
+            </Header>
+            <p>
+              Your participants will receive an e-mail asking them to register to their preference.
+              Once the Newdle is created, you will be shown a link you can share with anyone else
+              you wish to invite.
+            </p>
+          </div>
+          <div className={styles['create-button']}>
+            <Button
+              color="violet"
+              type="submit"
+              onClick={async () => {
+                const results = await createNewdle();
+                setNewdle(results);
+                setSuccessMessage(true);
+              }}
+            >
+              Create your Newdle!{' '}
+              <span role="img" aria-label="Steaming Bowl Emoji">
+                &#x1f35c;
+              </span>
             </Button>
-            <Button color="violet" icon labelPosition="right" onClick={() => dispatch(setStep(3))}>
-              Next step
-              <Icon name="caret right" />
+          </div>
+        </>
+      ) : (
+        <>
+          <Header as="h1" className={styles['newdle-title']}>
+            {newdle.title}
+          </Header>
+          <div className={styles['success-message']}>
+            <Header as="h3" className={styles['header']}>
+              Done!
+            </Header>
+            <p>
+              Your Newdle was created and invite e-mails have been sent. You can send the following
+              link to everyon you'd like to invite:
+            </p>
+            <div className={styles['newdle-link']}>{newdle.url}</div>
+          </div>
+          <div className={styles['summary-button']}>
+            <Button color="teal">
+              Go to Newdle summary{' '}
+              <span role="img" aria-label="Steaming Bowl Emoji">
```

Comment:
Do we really need that stuff instead of just putting the literal emoji there? It doesn't even reduce a11y since someone not seeing the emoji would not lose any relevant information.